RecoTracker/IterativeTracking/python/InitialStep_cff.py

```diff
@@ -290,14 +293,20 @@
 initialStep = ClassifierMerger.clone()
 initialStep.inputClassifiers=['initialStepClassifier1','initialStepClassifier2','initialStepClassifier3']
 
-#LWTNN selector
+trackingPhase1.toReplaceWith(initialStep, initialStepClassifier1.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorInitialStep_Phase1'),
+     src = 'initialStepTracks',
+     qualityCuts = [-0.95,-0.85,-0.75]
+))
+(trackingPhase1 & fastSim).toModify(initialStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
```

Comment:
this is not needed and may be a bit confusing re pp_on_AA below: the fastsim value is inherited from L281 and this line can be removed

Reply:
Removed the redundant line